src/EditorFeatures/CSharpTest/Diagnostics/DiagnosticAnalyzerDriver/DiagnosticAnalyzerDriverTests.cs

```diff
@@ -50,6 +50,9 @@ public async Task DiagnosticAnalyzerDriverAllInOne()
             // https://github.com/dotnet/roslyn/issues/44682 - Add to all in one
             missingSyntaxNodes.Add(SyntaxKind.WithExpression);
             missingSyntaxNodes.Add(SyntaxKind.RecordDeclaration);
+            // PROTOTYPE
+            missingSyntaxNodes.Add(SyntaxKind.RoleDeclaration);
```

Comment:
It looks like the AllInOne does contain these?

Reply:
The test didn't pass despite AllInOne change. Didn't spend time to investigate given this is in analyzer area